cmd/lncli/main.go

```diff
@@ -276,6 +293,11 @@ func main() {
 			Usage:     "The path to lnd's base directory.",
 			TakesFile: true,
 		},
+		cli.StringFlag{
+			Name:  "socksproxy",
+			Value: defaultLndDir,
+			Usage: "The host:port of a SOCKS proxy through which all connections to the LN daemon will be established over.",
```

Comment:
nit: break line after 80 characters (counting the tab character as 8 spaces).

Reply:
@guggero I've updated it according to the other `cli.StringFlag` options. Please let me know if that looks good.